Selenium.Axe/Selenium.Axe/HtmlReport.cs

```diff
@@ -425,8 +426,40 @@ private static void GetReadableAxeResults(AxeResultItem[] results, string type,
 
                     htmlAndSelector.InnerHtml = content.ToString();
                     htmlAndSelectorWrapper.AppendChild(htmlAndSelector);
+
+                    // Add fixes if this is for violations
+                    if (ResultType.Violations.Equals(type) && item.Any.Length > 0)
```

Comment:
If you're going to include `Any` content, you probably want to additionally include `All` and `None` content, which have similar how-to-fix messages. Most rules use just one of the 3 fix types, but there are some rules that use a mix (eg, "fix all of <`All` items> plus one of <`Any` items>"). The json files in https://github.com/dequelabs/axe-core/tree/develop/lib/rules define the relationships between each rule and specific all/any/none checks, if you'd like to look up examples of rules to test against

Reply:
Updated